pages/index.vue

```diff
@@ -367,7 +367,7 @@ export default {
         dialog: false,
         localuser:'',
         userAffiliation:[],
-        userPosition:null,
+        userPosition:false,
```

Comment:
Can you confirm this works as intended and doesn't insert the word "false" as a default value into the textbox?  I'd thing that the default value should actually be a an empty string ("").  If we're trying to store a NULL in the database then we need to double check how it's passed to the backend when the field is not touched by the user.  Maybe you've checked this already (I just want to confirm).